src/features/room/api/useDeleteMemoRoomMutation.tsx

```diff
@@ -0,0 +1,23 @@
+import { useMutation, UseMutationOptions } from '@tanstack/react-query';
+
+import axios from '@src/shared/configs/axios';
+import { DeleteMemoRooms } from '@src/shared/types/api/memoRooms';
+import { queryClient } from '@src/shared/configs/react-query';
+import { memoRoomKeys } from '@src/shared/utils/queryKeys';
+
+export const deleteMemoRoom = async (id: number) => {
+  const res = await axios.delete<DeleteMemoRooms['res']>(`/memo-rooms/${id}`);
+  return res.data;
+};
+
+const useDeleteMemoRoomMutation = (
+  options?: UseMutationOptions<DeleteMemoRooms['res'], unknown, { id: number }>,
+) => {
+  return useMutation(({ id }: { id: number }) => deleteMemoRoom(id), {
+    onSuccess: () => {
+      queryClient.invalidateQueries(memoRoomKeys.list());
+    },
+    ...options,
+  });
+};
```

Comment:
```suggestion
export const deleteMemoRoom = async ({ id }: { id: number }) => {
  const res = await axios.delete<DeleteMemoRooms['res']>(`/memo-rooms/${id}`);
  return res.data;
};

const useDeleteMemoRoomMutation = (
  options?: UseMutationOptions<DeleteMemoRooms['res'], unknown, { id: number }>,
) => {
  return useMutation({
    mutationFn: deleteMemoRoom,
    onSuccess: () => {
      queryClient.invalidateQueries(memoRoomKeys.list());
    },
    ...options,
  });
};
export default useDeleteMemoRoomMutation;
``` 
따로 화살표함수 쓰지말고 이런식으로 선언하는건 어떠신가요?

Reply:
react hook form 적용하면서 수정했습니다!
[f870c4d](https://github.com/memochat/memochat-client/pull/45/commits/f870c4ddd933eeca9676593bff528af885c09e6b)